lib/loginUtils.js

```diff
@@ -11,28 +11,31 @@ export async function loadStoreData() {
 export async function lookupClerk(storeId, pin) {
   const currStore = await getStoresById(storeId);
   const clerks = await getClerksByIds(currStore.clerkIds);
-  let errorMsg = 'Invalid PIN';
+  let errorMsg = '';
```

Comment:
I think we defaulted to 'Invalid PIN' as the error message because we don't necessarily want to expose the other case to the user, but not really sure